HW_1/Task4.py

```diff
@@ -0,0 +1,21 @@
+'''
+Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, 
+если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+*Пример:*
+6 -> 1  4  1
+24 -> 4  16  4
+60 -> 10  40  10
+'''
+s = int(input('Введите количество сделанных журавликов: '))
+
+if int(s % 6 == 0):
+    # p = int(s/6)
+    # c =int(s/6)
+    # k = int(p*4)
+    # print(f'{s} -> {p}, {k}, {c}')
+    print(f'{s} -> {int(s / 6)}, {int(s / 3 * 2)}, {int(s / 6)}')
```

Comment:
Тут больше подходит оператор целочисленного деления.
```suggestion
    print(f'{s} -> {s // 6}, {s // 3 * 2}, {s // 6}')
```